### PR TITLE
Milestone: Create conference page

### PR DESCRIPTION
**Pull request by Timbar09**
In this PR, I have created a conference page. The changes made are on the milestone PR numbers below.

**Changes**

#1 
#2 
#3 
#4

### CODE REVIEW

File: pages/about.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+              <li>
+                <a href="#about-home" class="header__links-link">About</a>
+              </li>
+
+              <li>
+                <a href="#partners" class="header__links-link">sponsor</a>
+              </li>
+
+              <li><a href="#news" class="header__links-link">News</a></li>
+            </ul>
```
Comment (dieum512) [inline pages/about.html:38]: good job working on this links list :+1:  but kindly, add a program link to this list so I can take us back to the  program link

**Review by dieum512 — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+        <img class="footer__logo" src="images/logo-dark.svg" alt="logo" />
+
+        <div class="footer__info">
+          <p class="footer__copyright">2023 Africa's Leading Tech Conference. Some rights reserved</p>
+          <p class="footer__credits">
+            <a class="about__link" href="https://github.com/Timbar09" target="_blank">Credits</a>
+          </p>
+        </div>
+      </div>
+    </footer>
```
Comment (alexander16108) [inline index.html]: image
- as required your text should have a logo aligned to the left and text aligned to the right  😄 
- your background colour should be the same as the background colour of your home section

File: pages/about.html [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    <header class="header">
+      <section class="header__nav-section">
+        <div class="container container-px">
+          <nav class="header__nav flex-jc-sb flex-ai-c">
+            <button id="btnHumburger" class="header__hamburger flex fle-jc-c flex-ai-c flex-collumn">
+              <span class="header__hamburger-stroke header__hamburger-top"></span>
+              <span class="header__hamburger-stroke header__hamburger-middle"></span>
+              <span class="header__hamburger-stroke header__hamburger-bottom"></span>
+            </button>
+
```
Comment (alexander16108) [inline pages/about.html:38]: image
on your hamburger menu in the about section of your page's mobile version, your home link takes you to nowhere and returns an error kindly fix that please 👍🏾

File: pages/about.html [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+        <img class="footer__logo" src="../images/logo-dark.svg" alt="logo" />
+
+        <div class="footer__info">
+          <p class="footer__copyright">2023 Africa's Leading Tech Conference. Some rights reserved</p>
+          <p class="footer__credits"><a class="about__link" href="https://github.com/Timbar09">Credits</a></p>
+        </div>
+      </div>
+    </footer>
+  </body>
+</html>
```
Comment (alexander16108) [inline pages/about.html]: image
your design ⬆️ 

image
required design ⬆️ 

are required your about section should not contain partners' information which is already present on the home page and your footer should have a background colour of black.

File: pages/about.html [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+            </ul>
+
+            <ul class="header__menu flex flex-collumn">
+              <li class="header__menu-option header__menu-option-home">
+                <a href="/" class="header__menu-link">Home</a>
+              </li>
+              <li class="header__menu-option"><a href="#about-home" class="header__menu-link">About</a></li>
+              <li class="header__menu-option"><a href="#partners" class="header__menu-link">sponsor</a></li>
+              <li class="header__menu-option"><a href="#news" class="header__menu-link">News</a></li>
+            </ul>
```
Comment (alexander16108) [inline pages/about.html]: image
your design ⬆️ 

image
required design ⬆️ 

- as seen about your navigation should include the section with the social media icons and other details 😄 
- your padding around your nav bar should also be reduced as well as seen above

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @Timbar09 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
  Your project looks beautiful and amazing, however the are some final touches which need to be added before it can be called perfect

## Highlights

- All linters are passing ✔️
- Good commit history ✔️
- Your design looks good 💯 ✔️
- Good PR description ✔️

## Required Changes ♻️
_Check the comments under the review._

- [ ] your video should contain a piece of code you wrote that you're impressed with or you find proud of making

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Timbar09) [reply index.html]: Thanks for the feedback @alexander16108 

I don't understand what I should change here. Can you please show me an example?

File: pages/about.html (hunk 3 above)
Comment (Timbar09) [reply pages/about.html:38]: I got you @alexander16108

I have just fixed that.

File: pages/about.html (hunk 4 above)
Comment (Timbar09) [reply pages/about.html]: Got it.

File: pages/about.html (hunk 5 above)
Comment (Timbar09) [reply pages/about.html]: Got it.

**Comment by Timbar09** [comment]
@alexander16108 

All the code that I am showing in the video, is code that I am actually proud of.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: pages/about.html (hunk 3 above)
Comment (Abenezer-Tilahun) [reply pages/about.html:38]: Fixed.

File: pages/about.html (hunk 4 above)
Comment (Abenezer-Tilahun) [reply pages/about.html]: Fixed.

File: pages/about.html (hunk 5 above)
Comment (Abenezer-Tilahun) [reply pages/about.html]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Timbar09  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes . 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**